app/setting/members/page.tsx

```diff
@@ -40,7 +41,14 @@ const Members = () => {
   const handleClick = async (hoveredItem: string | null, userId: string) => {
     if (hoveredItem === 'Make Admin') {
       try {
-        if (userId) await makeAdmin({ userId, role: UserRole.ADMIN });
+        workspaceStore.setLoading(true);
+        if (userId) {
+          const result = await makeAdmin({ userId, role: UserRole.ADMIN });
+          if (result) {
+            setUpdateRole(!updateRole);
```

Comment:
Instead of state change call this directly
getWorkspaceMember()

Reply:
Done